src/utils/argon-util.ts

```diff
@@ -0,0 +1,19 @@
+import * as argon2 from 'argon2';
+
+const verifyToken = async (payload: string, token: string) => {
```

Comment:
```suggestion
const verifyToken =  (payload: string, token: string) => {
```

Não precisa do async.